packages/logs/README.md

```diff
@@ -1,6 +1,6 @@
 # Browser Log Collection
 
-Send logs to Datadog from web browsers or other Javascript clients with the browser logs SDK.
+Send logs to Datadog from web browsers with the browser logs SDK.
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** to avoid misleading support for web extension, web worker.‏
```suggestion
Send logs to Datadog from browser web pages with the browser logs SDK.
```